### ISSUE TITLE
Point to another generated file type?

### ISSUE THREAD

**Issue by vicky-carbon** [zhouzi/graphql-codegen-factories#19]
I generated my typescript types in one file, but I want to generate these factories in a different file. Is it possible to add a config to define where to import these types from?
```yml
generates:
  ./src/testing/__generated__/factories.ts:
    plugins:
      - graphql-codegen-factories
    config:
      factoryName: create{Type}
  ./src/graphql/types/__generated__/type.ts:
    plugins:
      - typescript
    config:
      scalars:
        DateTime: string
```
Doing the above generates the functions fine, but TS complains because it can't find the type.
<img width="473" alt="Screen Shot 2022-02-21 at 6 03 36 PM" src="https://user-images.githubusercontent.com/82122118/155036134-8779cb7a-207f-408f-b80e-d92e62c70998.png">

**Comment by zhouzi** [comment]
Thanks for reporting @vicky-carbon 🙌 I just published `graphql-codegen-factories@1.0.0-beta.0` which adds `config.typesPath` for that purpose. Have a look at the [documentation](https://github.com/zhouzi/graphql-codegen-factories#configtypespath) and the [example](https://github.com/zhouzi/graphql-codegen-factories/tree/main/examples/splitted-output).

Feel free to give it a try and let me know 😉

**Comment by vicky-carbon** [comment]
I finally got around to trying it. It works! Thank you.
